tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -574,7 +574,11 @@ SpirvEmitter::getInterfacesForEntryPoint(SpirvFunction *entryPoint) {
       interfaces.insert(moduleVar);
     }
   }
-  return std::vector<SpirvVariable *>(interfaces.begin(), interfaces.end());
+  std::vector<SpirvVariable *> interfacesInVector;
```

Comment:
Since you know the final size of the vector, can you add:
```
interfacesInVector.reserve(interfaces.size());
```

so that you don't have to resize the vector?

Reply:
done